sql/sql_parse.cc

```diff
@@ -1274,7 +1274,6 @@ bool dispatch_command(THD *thd, const COM_DATA *com_data,
     the slow log only if opt_log_slow_admin_statements is set.
   */
   thd->enable_slow_log= TRUE;
-  thd->clear_slow_extended();
```

Comment:
It is redundant because of the same call in reset_for_next_command() ?

Reply:
Yes

Reply:
This was not redundant after all and caused PS-5570. Re-adding this line back, now with comment